test/api/package/get-supporting-companies.test.js

```diff
@@ -0,0 +1,296 @@
+const test = require('ava')
+const { ObjectId } = require('mongodb')
+const { before, beforeEach, afterEach, after } = require('../../_helpers/_setup')
+
+test.before(async (t) => {
+  await before(t, async ({ db, auth }) => {
+    const { id: userId1 } = await db.user.create({ email: 'honey@etsy.com', username: 'peter' })
+    t.context.userId1 = userId1.toHexString()
+
+    const session = await auth.user.createWebSession({ userId: t.context.userId1 })
+    t.context.sessionId = session.sessionId
```

Comment:
Since this is an unauth'd route I don't think this stuff is necessary. (It looks like userId is used in one of the package's maintainers list, but that isn't necessary for testing this route either).